src/org/usfirst/frc4904/autonly/Field.java

```diff
@@ -1,25 +1,83 @@
 package org.usfirst.frc4904.autonly;
 
 
+import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.DriverStation.Alliance;
 
 public class Field {
 	public Piece ourSwitch;
 	public Piece scale;
 	public Piece theirSwitch;
-	public Alliance ourTeam;
-
+	public Alliance team;
+	
+	/** The full constructor of the field.
```

Comment:
The JavaDocs are really good, however it's the norm to not start the documentation on the first line. Most people opt for something like:
```java
/**
 * The full constructor of the field.
 *
 * @param
 * @param
 */
```